packages/components/select-v2/src/useProps.ts

```diff
@@ -18,7 +18,7 @@ export const defaultProps: Required<Props> = {
   options: 'options',
 }
 
-export function useProps(props: Pick<ISelectV2Props, 'props'>) {
+export function useProps(props: ISelectV2Props) {
```

Comment:
It seems that there is no need to change here. Only `props.props` is used below, and only the corresponding types are extracted.

Reply:
I think the type should match the runtime value. Anyway, it doesn't matter. I'm fine with both.